### ISSUE TITLE
Runtime exception on player connection

### ISSUE THREAD

**Issue by SlimeDog** [danirod12/NTD-LuckyBlock#10]
```
> version
[10:02:17 INFO]: This server is running Paper version 1.21.3-10-master@92131ad (2024-11-01T21:30:00Z) (Implementing API version 1.21.3-R0.1-SNAPSHOT)
You are running the latest version
Previous version: 1.21.3-9-d19be64 (MC: 1.21.3)
[10:02:29 INFO]: rambeau lost connection: Disconnected
[10:02:29 INFO]: rambeau left the game
[10:03:29 INFO]: Server empty for 60 seconds, pausing
> version
[10:09:43 INFO]: This server is running Paper version 1.21.3-10-master@92131ad (2024-11-01T21:30:00Z) (Implementing API version 1.21.3-R0.1-SNAPSHOT)
You are running the latest version
Previous version: 1.21.3-9-d19be64 (MC: 1.21.3)
> version ntdLuckyBlock
[10:09:50 INFO]: ntdLuckyBlock version 2.9.10
[10:09:50 INFO]: Cool revamped LuckyBlock & API. Say goodbye to the sponge and resourcepacks!
[10:09:50 INFO]: Author: danirod12
```

ntdLuckyBlock loaded and enabled successfully:
```
[09:35:27] [Paper Plugin Remapper Thread - 2/INFO]: [PluginRemapper] Remapping plugin 'plugins/ntdLuckyBlock-2.9.10.jar'...
[09:35:27] [Paper Plugin Remapper Thread - 2/INFO]: [PluginRemapper] Done remapping plugin 'plugins/ntdLuckyBlock-2.9.10.jar' in 385ms.
[09:36:08] [Server thread/INFO]: [ntdLuckyBlock] Loading server plugin ntdLuckyBlock v2.9.10
[09:36:59] [Server thread/INFO]: [ntdLuckyBlock] Enabling ntdLuckyBlock v2.9.10
[09:36:59] [Server thread/INFO]: [ntdLuckyBlock] Thanks for installing PREMIUM plugin version!
[09:36:59] [Server thread/INFO]: [ntdLuckyBlock] Starting LuckyBlock (ntdLuckyBlock) v2.9.10, build133(28/10/2024), premium
[09:36:59] [Server thread/INFO]: [ntdLuckyBlock] user: 56363, resource ID: 94872, nonce: 67784202
[09:36:59] [Server thread/INFO]: [ntdLuckyBlock] 
[09:36:59] [Server thread/INFO]: [ntdLuckyBlock] =-= SUPPORT § BUG REPORTING § FEATURE REQUESTING =-=
[09:36:59] [Server thread/INFO]: [ntdLuckyBlock] =-= SUPPORT § BUG REPORTING § FEATURE REQUESTING =-=
[09:36:59] [Server thread/INFO]: [ntdLuckyBlock] Loaded main material version
[09:36:59] [Server thread/INFO]: [ntdLuckyBlock] Loading NMS for your platform (1.21.3-10-92131ad (MC: 1.21.3), v1_21_R2)...
[09:36:59] [Server thread/INFO]: [ntdLuckyBlock] Loading language file "en" by DenBeKKer
[09:36:59] [Server thread/INFO]: [ntdLuckyBlock] "Web-Server is unavailable" message is disabled. Plugin page - https://www.spigotmc.org/resources/94872
[09:36:59] [Server thread/INFO]: [ntdLuckyBlock] Vault connected
[09:36:59] [Server thread/INFO]: [ntdLuckyBlock] WorldEdit connected
[09:36:59] [Server thread/INFO]: [ntdLuckyBlock] WorldGuard connected
[09:36:59] [Server thread/INFO]: [ntdLuckyBlock] BedWars1058 not found
[09:36:59] [Server thread/INFO]: [ntdLuckyBlock] MBedwars not found
[09:36:59] [Server thread/INFO]: [ntdLuckyBlock] CoreProtect not found
[09:36:59] [Server thread/INFO]: [ntdLuckyBlock] TokenManager not found
[09:36:59] [Server thread/INFO]: [ntdLuckyBlock] SlimeFun not found
[09:36:59] [Server thread/INFO]: [ntdLuckyBlock] MMOItems not found
[09:36:59] [Server thread/INFO]: [ntdLuckyBlock] MythicMobs not found
[09:36:59] [Server thread/INFO]: [ntdLuckyBlock] Found 3/10 compatible plugins
[09:36:59] [Server thread/INFO]: [ntdLuckyBlock] Loading system...
[09:36:59] [Server thread/INFO]: [ntdLuckyBlock] Loaded 0 internal custom items...
[09:36:59] [Server thread/INFO]: [ntdLuckyBlock] Loaded 0 custom items... (Total)
[09:36:59] [Server thread/INFO]: [ntdLuckyBlock] Loaded 2 luckyblock types...
[09:36:59] [Server thread/INFO]: [ntdLuckyBlock] System loaded (took 42 ms)...
[09:36:59] [Server thread/INFO]: [ntdLuckyBlock] Enabled (took 214 ms)... 
```
But on player connection (in a world that contains no luckyblocks):
```
[10:02:12 ERROR]: Could not pass event PlayerInteractEvent to ntdLuckyBlock v2.9.10
java.lang.RuntimeException: Not supported on 1.20.5+, use ItemLoader instead
        at ntdLuckyBlock-2.9.10.jar/me.DenBeKKer.ntdLuckyBlock.e.S.int(Unknown Source) ~[ntdLuckyBlock-2.9.10.jar:?]
        at ntdLuckyBlock-2.9.10.jar/me.DenBeKKer.ntdLuckyBlock.e.e.do(Unknown Source) ~[ntdLuckyBlock-2.9.10.jar:?]
        at ntdLuckyBlock-2.9.10.jar/me.DenBeKKer.ntdLuckyBlock.customitem.CustomItemFactory.parseValue(Unknown Source) ~[ntdLuckyBlock-2.9.10.jar:?]
        at ntdLuckyBlock-2.9.10.jar/me.DenBeKKer.ntdLuckyBlock.customitem.CustomItemFactory.fetchCustomItem(Unknown Source) ~[ntdLuckyBlock-2.9.10.jar:?]
        at ntdLuckyBlock-2.9.10.jar/me.DenBeKKer.ntdLuckyBlock.d.b.do(Unknown Source) ~[ntdLuckyBlock-2.9.10.jar:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor582.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:77) ~[paper-api-1.21.3-R0.1-SNAPSHOT.jar:?]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[paper-api-1.21.3-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[paper-api-1.21.3-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54) ~[paper-1.21.3.jar:1.21.3-10-92131ad]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:131) ~[paper-1.21.3.jar:1.21.3-10-92131ad]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:628) ~[paper-api-1.21.3-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:603) ~[paper-1.21.3.jar:1.21.3-10-92131ad]
        at org.bukkit.craftbukkit.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:559) ~[paper-1.21.3.jar:1.21.3-10-92131ad]
        at org.bukkit.craftbukkit.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:554) ~[paper-1.21.3.jar:1.21.3-10-92131ad]
        at org.bukkit.craftbukkit.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:550) ~[paper-1.21.3.jar:1.21.3-10-92131ad]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.handleUseItem(ServerGamePacketListenerImpl.java:2028) ~[paper-1.21.3.jar:1.21.3-10-92131ad]
        at net.minecraft.network.protocol.game.ServerboundUseItemPacket.handle(ServerboundUseItemPacket.java:47) ~[paper-1.21.3.jar:1.21.3-10-92131ad]
        at net.minecraft.network.protocol.game.ServerboundUseItemPacket.handle(ServerboundUseItemPacket.java:10) ~[paper-1.21.3.jar:1.21.3-10-92131ad]
        at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$0(PacketUtils.java:55) ~[paper-1.21.3.jar:1.21.3-10-92131ad]
        at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.21.3.jar:1.21.3-10-92131ad]
        at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:154) ~[paper-1.21.3.jar:1.21.3-10-92131ad]
        at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[paper-1.21.3.jar:1.21.3-10-92131ad]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1576) ~[paper-1.21.3.jar:1.21.3-10-92131ad]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:207) ~[paper-1.21.3.jar:1.21.3-10-92131ad]
        at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:128) ~[paper-1.21.3.jar:1.21.3-10-92131ad]
        at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1553) ~[paper-1.21.3.jar:1.21.3-10-92131ad]
        at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1546) ~[paper-1.21.3.jar:1.21.3-10-92131ad]
        at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:138) ~[paper-1.21.3.jar:1.21.3-10-92131ad]
        at net.minecraft.server.MinecraftServer.managedBlock(MinecraftServer.java:1497) ~[paper-1.21.3.jar:1.21.3-10-92131ad]
        at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1507) ~[paper-1.21.3.jar:1.21.3-10-92131ad]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1343) ~[paper-1.21.3.jar:1.21.3-10-92131ad]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:339) ~[paper-1.21.3.jar:1.21.3-10-92131ad]
        at java.base/java.lang.Thread.run(Thread.java:1575) ~[?:?]
```

**Comment by danirod12** [comment]
Find difference
![image](https://github.com/user-attachments/assets/4509d9b1-c52d-42c4-8aad-241dbbb819e2)

Thank you for reporting, will release an update now

**Comment by danirod12** [comment]
released an update 2.9.11

**Comment by SlimeDog** [comment]
Confirmed fixed with ntdLuckyBlock 2.9.11. Thanks for fast response.
